website/docs/d/is_lbs.html.markdown

```diff
@@ -23,6 +23,7 @@ Review the attribute references that you can access after you retrieve your data
 - `load_balancers` - (List) The Collection of load balancers.
 
   Nested scheme for `load_balancers`:
+    - `access_tags`  - (String) Access management tags associated for the load balancers.
```

Comment:
use load balancer in description instead of load balancers, this will apply for other List data-resources.